app/manager.py

```diff
@@ -41,8 +41,8 @@ def create_db():
     db.create_all()
 
     # Load room definition
-    roomConfigs = [os.path.join(folder, 'rooms.json') for folder in [configdir, overridedir]]
-    roomConfig = make_merged_cfg(roomConfigs)['rooms']
+    roomConfigPaths = [os.path.join(folder, 'rooms.json') for folder in [configdir, overridedir]]
+    roomConfig = load_and_validate_rooms(roomConfigPaths)
```

Comment:
It occurred to me that this might be redundant, since [the application context is initialized inside `manager.py`](https://github.com/morganecf/jitsi-party/blob/master/app/manager.py#L12). This instantiates a config object, whose constructor already calls `load_and_validate_rooms`. We should be able to reference `app.config['ROOMS']` to populate the db with rooms, instead of calling `load_and_validate_rooms` for the second time.

Reply:
You're right! Just tested it and it seems to work fine. Once I ran `flask create-db` I could query all the rooms from the flask shell, and when I ran & connected to the app it all seemed to work as expected.

Reply:
Cool!